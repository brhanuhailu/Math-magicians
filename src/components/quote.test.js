import renderer from 'react-test-renderer';
import Quote from './quote';

describe('Test Quotes component', () => {
  it('Test component should render properly', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
