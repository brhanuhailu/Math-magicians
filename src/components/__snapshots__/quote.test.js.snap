// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Quotes component Test component should render properly 1`] = `
<div
  className="qoutes"
>
  <h1>
     Quotes from Api about happiness
  </h1>
</div>
`;
